src/main/java/seedu/address/model/schedule/Schedule.java

```diff
@@ -9,7 +9,7 @@
 
 public class Schedule {
 
-    public static final String EMPTY_SCHEDULE_MESSAGE = "No schedule recorded yet.";
+    public static final String EMPTY_SCHEDULE_MESSAGE = "No schedule recorded for %1$s.";
```

Comment:
The `%1$s` is meant to pass the `Person`'s name right? In that case I think this message should not be in the `Schedule` class, but rather in the `Person` class, otherwise it breaks abstraction.

Reply:
Alright, will change